chromium_src/third_party/blink/renderer/core/execution_context/execution_context.cc

```diff
@@ -147,7 +145,8 @@ AudioFarblingCallback BraveSessionCache::GetAudioFarblingCallback(
       }
       case BraveFarblingLevel::MAXIMUM: {
         uint64_t seed = *reinterpret_cast<uint64_t*>(domain_key_);
-        return base::BindRepeating(&PseudoRandomSequence, seed);
+        auto prng = std::make_shared<std::mt19937_64>(std::mt19937_64(seed));
```

Comment:
a quick note: this is also currently banned in Chromium codebase. https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++11.md#Shared-Pointers

Reply:
Of course. Thanks for pointing that out. Maybe I can get `base::Owned` to work...

Reply:
`base::OwnedRef` seems to work.